regulatory_compliance/storage.sp

```diff
@@ -376,17 +376,11 @@ query "storage_account_block_public_access" {
     select
       sa.id as resource,
       case
-        when sa.id not like '%/resourceGroups/aro-%'
-          and (sa.name not like 'cluster%' or sa.name not like 'imageregistry%')
-          and sa.allow_blob_public_access = 'false'
-          then 'ok'
+        when sa.public_network_access = 'Disabled' then 'ok'
         else 'alarm'
       end as status,
       case
-        when sa.id not like '%/resourceGroups/aro-%'
-          and (sa.name not like 'cluster%' or sa.name not like 'imageregistry%')
-          and sa.allow_blob_public_access = 'false'
-          then sa.name || ' not publicy accessible.'
+        when sa.public_network_access = 'Disabled' then sa.name || ' not publicy accessible.'
```

Comment:
@khushboo9024 would it be wise to check for `disabled` as well, since we cannot guarantee the response will remain consistent?